libs/features/mas/test/hydrate.test.js

```diff
@@ -476,6 +476,11 @@ describe('hydrate', () => {
 
     it('should hydrate a ccd-slice merch card', async () => {
         const fragment = {
+            settings: {
+                stockCheckboxLabel: '{{stock-checkbox-label}}',
+                stockOfferOsis: '',
```

Comment:
since `stockOfferOsis` is an array, I wonder what is the best to author it in MAS Studio and pass it down to Milo.
Looking at https://github.com/adobecom/milo/blob/stage/libs/features/mas/src/merch-card.js#L76, it is supposed to be 3 osis comma separated.
If we author those OSIs seperately, we might need to stitch them together during the hydration process.